configs/configs.go

```diff
@@ -55,3 +61,51 @@ func LoadConfig(path string, obj interface{}) error {
 	}
 	return nil
 }
+
+// GLogFlags contains the values of the flags that github.com/golang/glog
+// registers.
+type GLogFlags struct {
```

Comment:
Suggestion: I'd be tempted to remove call this "CommonFlags" (or something in that spirit), so if we add non-glog flags to it later we won't have to rename it. (Ditto for SetGlogFlags).